### ISSUE TITLE
Missing Support 'Non ASCII Characters' in Pygments1.6

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#569]
(Original issue 862 created by kashewnuts on 2013-03-26T15:50:37.756726+00:00)

* I  wanto to turn off red frame or go back to Pygments1.5's layout.
* I updated Pygments from 1.5 to 1.6.
* Then, red border around the 'Non ASCII Characters' in Pygments1.6.
* pygments_style is 'emacs'.
* Not only 'Non ASCII Characters', but it also sign like " *$ [  '* ".

* Original source is following.

```
#!guess

.. code-block:: guess

    $ pip install tinkerer==0.4.1b (Tinkererのインストール)
    $ mkdir myblog
    $ cd myblog
    $ tinkerer -s (Tinkererの初期セットアップ)

```

**Comment by Anteru** [comment]
(Original issue 862 last updated on 2013-05-20T05:44:34.819983+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2013-05-19T09:38:16.487088+00:00)

I guess that the "guessed" lexer is the wrong one in Pygments 1.6. Can you use an explicit "bash" instead of "guess"?

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2013-05-20T05:44:34.814870+00:00)

Closes #862: better guess for shell lexer.

→ <<cset e586a499a996>>
